csharp/test/Drivers/Apache/Spark/StringValueTests.cs

```diff
@@ -50,11 +50,11 @@ public static IEnumerable<object[]> ByteArrayData(int size)
         [InlineData(null)]
         [InlineData("")]
         [InlineData("你好")]
-        [InlineData("String contains formatting characters tab\t, newline\n, carriage return\r.", "3.4.0")]
+        [InlineData("String contains formatting characters tab\t, newline\n, carriage return\r.", SparkServerType.Databricks)]
         [InlineData(" Leading and trailing spaces ")]
-        public async Task TestStringData(string? value, string? minVersion = null)
+        internal async Task TestStringData(string? value, SparkServerType? serverType = default)
```

Comment:
Will these tests still run if marked as `internal`? (Some test frameworks require `public`.)

Reply:
Yes. No problem with this framework. A side effect of making `SparkServerType` internal.